### ISSUE TITLE
[a11y]: ai-chat should not announce state of microphone button twice

### ISSUE THREAD

**Issue by Daniel-Schulz-Private** [carbon-design-system/carbon-labs#878]
### Package version

0.18.0

### React version

18.3.1

### Automated testing tool and ruleset

n/a

### Assistive technology

JAWS, Voice Over

### Description

Dual status announcement observed:
When the Microphone button (at the right hand side of the question input field) is in a disabled state, screen readers announce the button label along with its disabled status. Because the label itself contains the word “Unavailable,” users hear duplicate status information, creating a confusing experience.

• Example announcement: “Microphone Button Unavailable Unavailable.”

Recommended fix: Remove the word “Unavailable” from the button label. Instead, use a clear, meaningful label such as “Microphone” and rely on the programmatic disabled
state (aria-disabled or disabled attribute) to communicate the button’s availability to assistive technologies.

User impact: Impact Screen reader users
Adding status indication wordings to labels makes it harder to understand the actual status of the control

<img width="1435" height="458" alt="Image" src="https://github.com/user-attachments/assets/c0536db9-948b-4ee5-807c-3350ce24db99" />

### WCAG 2.1 Violation

WCAG Mapping: 4.1.2 Name, Role, Value

### Reproduction/example

n/a

### Steps to reproduce

1. make sure JAWS is active
2. open ai-chat
3. have microphone button announced with JAWS
4. see, that state "unavailable" is announced twice

### Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon-labs/blob/main/.github/CODE_OF_CONDUCT.md)
- [x] I checked the [current issues](https://github.com/carbon-design-system/carbon-labs/issues) for duplicate problems
